osi_detectedlandmark.proto

```diff
@@ -375,4 +385,82 @@ message CandidateRoadMarking
     // 
     // Unit: [s]
     optional double age = 3;
+
+    // The measurement state.
+    //
+    optional MeasurementState measurement_state = 4;
+}
+
+//
+// \brief A landmark in the environment as detected by the sensor.
+//
+message DetectedLandmark
+{
+    // Specific ID of the landmark as assigned by the sensor internally.
+    // Need not match with \c #ground_truth_id_list.
+    //
+    optional Identifier tracking_id = 1;
+
+    // The ID of the original landmark in the ground truth.
+    // In case of a ghost detection (no corresponding ground truth), this field
+    // should be unset.
+    //
+    repeated Identifier ground_truth_id_list = 2;
+
+    // A list of candidates for this landmark as estimated by the sensor.
+    //
+    repeated CandidateLandmark candidate_landmarks = 3;
+
+    // The estimated probability that this landmark really exists, not based
+    // on history.
+    //
+    // \note Use as confidence measure where a low value means less confidence
+    // and a high value indicates strong confidence.
+    //
+    optional double existence_probability = 4;
+    
+    // The root mean squared error of the base parameters of the detected 
+    // landmark. \c RoadMarking::base has to be identical for
+    // all \c #candidate_landmarks landmarks.
+    //
+    optional BaseStationary base_rmse = 5;
+   
+    // A list of sensors which detected this detected entity.
+    //
+    // If \c SensorData has detected entities and all detections are missing, 
+    // then e.g. the number of sensors can confirm the #existence_probability.
+    // 
+    // \note This information can be determined via the detected entities'
+    // detections ( \c ...Detection::object_id = 'this detected entity' ) and
+    // the sensors (their IDs) to which these detections belong.
+    //
+    repeated Identifier sensor_id_list = 6;    
+}
+
+//
+// \brief A candidate for a detected landmark as estimated by the sensor.
+//
+message CandidateLandmark
+{
+    // The description of the road marking.
+    //
+    optional RoadMarking road_marking = 1;
```

Comment:
This should be `optional Landmark landmark` , since we describe `CandidateLandmark` here, shouldn't it?
The comment should be changed accordingly.

Reply:
Yes. Thx. I made a bugfix.